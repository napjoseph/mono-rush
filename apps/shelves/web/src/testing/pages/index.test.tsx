import React from 'react';
import { render } from '../testUtils';
import HomePage from '../../pages/index.page';

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<HomePage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
