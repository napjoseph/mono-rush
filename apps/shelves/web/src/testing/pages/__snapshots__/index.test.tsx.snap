// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="title"
  >
    Home Page
  </h1>
</DocumentFragment>
`;
